sdk/azidentity/credential_log_classification.go

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package azidentity
+
+import (
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+)
+
+const (
+	LogCredential        azcore.LogClassification = "credential"
+	LogCredentialVerbose azcore.LogClassification = "credentialVerbose"
```

Comment:
Other than one test this isn't used anywhere, can it be removed?  It can always be added later if needed.

Reply:
So I was keeping it because I would really like to log scopes with the more verbose classification instead of the main logCredential classification. But, for now, I think it's ok to leave it all in LogCredential since there isn't more information that could also be added to the verbose level.

Reply:
Removed.